kolibri/plugins/facility/assets/src/views/DataPage/index.vue

```diff
@@ -284,12 +290,33 @@
 
   @import '~kolibri-design-system/lib/styles/definitions';
 
-  .infobox {
-    padding: 8px;
-    margin-right: -8px;
-    margin-left: -8px;
-    font-size: 0.8em;
-    border-radius: $radius;
+  .subheading {
+    display: inline-block;
+    margin: 0 0.313rem 0.313rem 0;
+  }
+
+  .subheading-buttons {
+    display: inline-block;
+    margin-right: 0.313rem;
+  }
+
+  .generated-time {
+    margin: 0.313rem 0;
+    color: #616161;
```

Comment:
Kolibri can be themed with different colors, so using tokenized colors is recommended for this. This color seems to match the `$themeTokens.annotation` color. See https://design-system.learningequality.org/colors/#tokens-annotation

Note that you'll need to use `$themeTokens.annotation` in the component template or JS, not within the CSS